### DISCUSSION TITLE
[Feature] Store generated thumbnails

### DISCUSSION THREAD

**Discussion by noriakisana** [immich-app/immich#8564] [Feature Request]
### The feature

It would be nice, if the app could store the thumbnails of the images - which are not on the mobile as originals - so that you can browse through them fast, even without internet connection. As Addition you could add when zooming to load the original in the background - as optional choice.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#9706
